### ISSUE TITLE
Limit ping functionality for new members

### ISSUE THREAD

**Issue by ankitsmt211** [Together-Java/TJ-Bot#883]
### Problem
We have had multiple situations where a new member shows up, they are in a hurry to find a solution to a problem and skip through the "bare minimum" rules that they should look at. Mostly it turns into multiple pings to "Helper, Expert" roles in general which further requies some attention from moderators, reminders and a couple of uncomfortable pings for existing members.

### Possible Solution
We find a way to make the ability to ping anyone or roles void by default for a certain timeperiod(upto a week maybe) when they get familiar with how server functions. We could also limit it to "first post in #questions" since they could possibly join and act after a week or so.

### Implementation Ideas
- Default role when joining
- Keep track whether time period has passed or if they made their first post
- Maybe an event listener on pings server wide, to check if its from certain role.

**Comment by christolis** [comment]
Has anybody started work on this? Is it still active? I'd also love to know if it's a priority issue :)

**Comment by ankitsmt211** [comment]
Hey @christolis i don't think anyone picked up this one yet, the issue is active yes but not high priority.

**Comment by KonstantineVashalomidze** [comment]
I intend to work on this. Would you mind assigning it to me?

**Comment by KonstantineVashalomidze** [comment]
It is kind of strange though that @Helper works on my test discord server channel and not in TJ server. Can you tell me why? 
I think that pinging feature is completely turned off somehow.

**Comment by Zabuzard** [comment]
Yes, it is handled in the discord server settings nowadays. Not needed anymore.
